src/lib.rs

```diff
@@ -64,6 +64,49 @@
 //! handle).  These functions, called `value_from_*` and `value_to_*`, will
 //! correctly (un)pickle these types.
 //!
+//! By default `(value_)from_reader` closes the input stream. It is possible
```

Comment:
I'd prefer this new chunk to go to the `from_reader` docs, it is a bit too specialized for the crate intro doc.